truffle/com.oracle.truffle.api.vm/src/com/oracle/truffle/api/vm/PolyglotEngine.java

```diff
@@ -1285,3 +1285,21 @@ protected String toString(TruffleLanguage language, TruffleLanguage.Env env, Obj
         }
     } // end of SPIAccessor
 }
+
+class PolyglotEngineSnippets {
+    abstract class YourLang extends TruffleLanguage<Object> {
+        public static final String MIME_TYPE = "application/my-test-lang";
+    }
+
+    public static PolyglotEngine initializeWithParameters() {
+        // @formatter:off
+        // BEGIN: com.oracle.truffle.api.vm.PolyglotEngineSnippets#initializeWithParameters
+        String[] args = {"--kernel", "Kernel.som", "--instrument", "dyn-metrics"};
+        PolyglotEngine.Builder builder = PolyglotEngine.newBuilder();
+        builder.config(YourLang.MIME_TYPE, "CMD_ARGS", args);
+        PolyglotEngine vm = builder.build();
+        // END: com.oracle.truffle.api.vm.PolyglotEngineSnippets#initializeWithParameters
+        // @formatter:on
+        return vm;
+    }
+}
```

Comment:
No newline at the end of the file. How come checkstyle does not complain about it? Is it not enabled for the truffle.api.vm project?